### PR TITLE
Docker python api > 2.0

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Update cleanup_docker_images.py to match latest (docker 2.4.2) API.
Attempt to address #155

### CODE REVIEW

File: master/jenkins_files/files/home/jenkins-slave/cleanup_docker_images.py [hunk 1]
```diff
@@ -113,14 +113,14 @@ def main():
                         help='The minimum age of items to clean up in days.')
     parser.add_argument('--min-hours', type=int, default=10,
                         help='The minimum age of items to clean up in hours, added to days.')
-    parser.add_argument('--docker-api-version', type=str, default='1.16',
+    parser.add_argument('--docker-api-version', type=str, default='1.30',
```
Comment (mikaelarguedas) [inline master/jenkins_files/files/home/jenkins-slave/cleanup_docker_images.py]: Note: this is what is on my workstation, may need to be updated according to what version the buildfarm machines are provisioned with

Comment (nuclearsandwich) [inline master/jenkins_files/files/home/jenkins-slave/cleanup_docker_images.py]: what's the docker version on your workstation?

Comment (mikaelarguedas) [inline master/jenkins_files/files/home/jenkins-slave/cleanup_docker_images.py]: 17.06.0-ce

**Comment by mikaelarguedas** [comment]
closing in favor of https://github.com/nuclearsandwich/buildfarm_deployment/pull/1

**Pull request closed without merging**